wzdx/raw_to_standard/__init__.py

```diff
@@ -0,0 +1 @@
+from wzdx.combination.combine_wzdx import find_overlapping_features_and_combine
```

Comment:
Do we need to be importing the combine functionality here?

Reply:
The idea here is to expose commonly used folders and methods when the package is installed, to make importing easier. I don't fully understand how this works though. I will look into this a little more and see what I should do